db.go

```diff
@@ -464,11 +503,15 @@ func (db *DB) parseDataFiles(dataFileIds []int) (err error) {
 	}
 
 	readEntriesFromFile := func() error {
+		defer func() {
+			_ = f.release()
+		}()
+
 		for {
 			entry, err := f.readEntry(off)
 			if err != nil {
 				// whatever which logic branch it will choose, we will release the fd.
-				_ = f.release()
+				// _ = f.release()
```

Comment:
The old release only something wrong. But I think we also release it after iteration.